tests/unit/slang-utils/create-parser.test.js

```diff
@@ -84,13 +84,15 @@ describe('inferLanguage', function () {
   }
 
   test('should use the latest successful version if the source has no pragmas', function () {
-    createParser(`pragma solidity 0.8.28;`, options);
+    // This is to create in memory the latest parser and review the behaviour
+    createParser(`pragma solidity ${latestSupportedVersion};`, options);
```

Comment:
Not blocking of course, but just FYI: the newly shipped `Languagefacts.inferSupportedVersions()` can now replace most of the custom implementation in `createParser()`...

Reply:
I'm aware of this but was very busy moving out of my old house, travelling to Europe, and for the next few weeks I'll be AFK. So it didn't feel healthy to add more stress with an extra feature.
When I return to society I'll tackle this one.

Reply:
Of course. Thanks for all the great work so far!